GCDWebServer/Core/GCDWebServerPrivate.h

```diff
@@ -92,6 +92,13 @@
 
 #define __GCDWEBSERVER_LOGGING_FACILITY_COCOALUMBERJACK__
 
+/**
+ * Whether async should be used by log messages, excluding error messages that are always sent sync.
+ **/
+#ifndef LOG_ASYNC_ENABLED
+    #define LOG_ASYNC_ENABLED YES
```

Comment:
I see. This cocoalumberjack change should be in a separate PR. If you take it out from this PR I can merge it. Thanks.